src/osx/cocoa/textctrl.mm

```diff
@@ -1033,26 +1137,47 @@ - (void)controlTextDidEndEditing:(NSNotification *)aNotification
         [m_textView setFont: font.OSXGetNSFont()];
 }
 
+static double PointsFromInternalUnits(int units, double unitsPerPoint)
+{
+  return (double)units / unitsPerPoint;
+}
+
+static int InternalUnitsFromPoints(double pts, double unitsPerPoint)
+{
+  return wxRound(pts * unitsPerPoint);
+}
+
+NSString * const wxNSParagraphUserDataAttributeName = @"wxParagraphUserData";
+
 bool wxNSTextViewControl::GetStyle(long position, wxTextAttr& style)
 {
-    if (m_textView && position >=0)
-    {
+    if ( m_textView && position >= 0 )
```

Comment:
I'll fix this myself, but, for the future changes, please avoid making whitespace-only changes together with other significant ones. TIA!

Reply:
Sorry, trying to follow the coding standards.